packages/server/lib/browsers/webdriver-classic/index.ts

```diff
@@ -0,0 +1,272 @@
+import debugModule from 'debug'
+// using cross fetch to make unit testing easier to mock
+import crossFetch from 'cross-fetch'
+
+const debug = debugModule('cypress:server:browsers:webdriver')
+
+type InstallAddOnArgs = {
+  extensionPath: string
+  isTemporary: boolean
+}
```

Comment:
```suggestion
type InstallAddOnArgs = {
  path: string
  temporary: boolean
}
```
simplifies the fn body of `installAddOn`

Reply:
updated in 721100a489. should make the cutover to [webdriver](https://www.npmjs.com/package/webdriver) cleaner too